docs/quickstarts/ratify-on-azure.md

```diff
@@ -108,28 +108,35 @@ Configure user-assigned managed identity and enable `AcrPull` role to the worklo
 
 ## Configure access policy for AKV 
 
-> Note: Azure Key Vault integration currently only support self signed certificate, if you are working with a certificate chain, please specify the public root certificate value inline using the [inline certificate provider](../reference/crds/certificate-stores.md#inline-certificate-provider).
-
 1. Set the environmental variable for Azure Key Vault URI.
 
     ```bash
     export VAULT_URI=$(az keyvault show --name ${AKV_NAME} --resource-group ${GROUP_NAME} --query "properties.vaultUri" -otsv)
     ```
 
-2. Import your own private key and certificates. You can import it on the portal as well.
+2. Ratify requires secret permissions to retrieve the public certificates for the entire certificate chain,
+ please set private keys to Non-exportable at certificate creation time to avoid security risk. Learn more about non-exportable keys [here](https://learn.microsoft.com/en-us/azure/key-vault/certificates/how-to-export-certificate?tabs=azure-cli#exportable-and-non-exportable-keys)
 
+Use get-default-policy to get a template for your certificate policy, please update `contentType`, `subject` to suit your certificate and most importantly update `exportable` to false. Save your policy into a local file.
+```bash
+az keyvault certificate get-default-policy  > akvpolicy.json
+```
+3. Import your own private key and certificates. You can import it on the portal as well.  
+ Specify the file path of the certificate policy created from step2 with the -p flag.
+    
     ```bash
     az keyvault certificate import \
     --vault-name ${AKV_NAME} \
     -n ${KEY_NAME} \
-    -f ${CERT_PATH}
```

Comment:
Are there any steps that walk the reader through creating a certificate? Or is that assumed knowledge or a prerequisite?

Reply:
I ve actually deleted the entire kv import step, customer should follow https://learn.microsoft.com/en-us/azure/container-registry/container-registry-tutorial-sign-build-push#build-and-sign-a-container-image from the prereq section of this quick start.